ng-dev/pr/merge/defaults/labels.ts

```diff
@@ -32,15 +33,25 @@ import {assertActiveLtsBranch} from './lts-branch';
  * @param releaseConfig Configuration for the release packages. Used to fetch
  *   NPM version data when LTS version branches are validated.
  */
-export async function getDefaultTargetLabelConfiguration(
-  api: GithubClient,
-  githubConfig: GitClientConfig,
-  releaseConfig: ReleaseConfig,
+export async function getTargetLabels(): Promise<TargetLabel[]>;
+export async function getTargetLabels(
+  _api?: GithubClient,
+  _config?: Partial<{github: GitClientConfig; release: ReleaseConfig}>,
+): Promise<TargetLabel[]>;
+export async function getTargetLabels(
+  _api?: GithubClient,
+  _config?: Partial<{github: GitClientConfig; release: ReleaseConfig}>,
 ): Promise<TargetLabel[]> {
-  const nextBranchName = getNextBranchName(githubConfig);
+  const api = _api || GitClient.get().github;
+
+  const config = _config || (getConfig() as any);
```

Comment:
We should use `??` instead of `||` for falling back (for `_api` as well). You could also inline the `GitClient.get().github` as default parameter IIUC. Maybe we should have a small local function here that does the same for the config (with the both assertions; so we do not need the weird `as any` cast here)

Reply:
While reading below where `getTargetLabels` is called, I think we should just always require people to pass a `GithubClient` and the `config`. That is less confusing and more readable. Thoughts?

Reply:
I initially had them as default parameters, but when it was a default parameter I ran into issues in the tests, but now they aren't happening so it must have been something else.

Reply:
I see, yeah I think just making them required is probably the best IMO.

Reply:
I am going to hold off on making them required as we are just moving the problem up a level in terms of the call stack by making it required here.  I think there is a larger refactoring that can address it.

Reply:
I think it would be expected to pass the config around, compared to always retrieving it through the single-ton, but it's true that this is probably just up to personal preference. 

Yeah, let's keep it as is and we can see how it evolves over time.